src/CreateGroup.js

```diff
@@ -1,11 +1,66 @@
 import React, { useEffect, useState } from "react";
+import {Link} from "react-router-dom";
 import "./App.css";
+import { NavBar } from "./components/NavBar";
+import { Button, Dialog, DialogTitle, DialogContent,
+         DialogContentText, Container, TextField } from "@material-ui/core";
+import { FileCopy } from "@material-ui/icons";
+
+function GroupDialog() {
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = ()=> {
+    setOpen(false);
+  };
+
+  return(
+    <div>
+      <Button onClick={handleClickOpen}>Create Group</Button>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        >
+        <DialogTitle>Group Created</DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            <div>
+              Room ID: 343 <Button><FileCopy /></Button>
```

Comment:
Was about to ask about how ur generating the roomID but 
 check this out when u do -> hardcodedhttps://stackoverflow.com/questions/28822054/firebase-how-to-generate-a-unique-numeric-id-for-key